packages/documentation/src/stories/internet-header/components/internet-header.stories.tsx

```diff
@@ -15,11 +15,12 @@ const meta: Meta<HTMLSwisspostInternetHeaderElement> = {
     },
     badges: [BADGE.STABLE],
     controls: {
-      exclude: ['config-proxy']
+      exclude: ['config-proxy'],
     },
   },
   args: {
-    'project': 'test'
+    project: 'test',
+    language: 'de',
```

Comment:
Custom config doesn't work if you are not providing a value with the language attribute.

According to the language prop description and types, it should be possible to not provide any value attribute: https://github.com/swisspost/design-system/blob/fix/e2e-internet-header-with-storybook-v7/packages/internet-header/src/components/post-internet-header/post-internet-header.tsx#L48-L51

By default, it will not load the custom config here:

https://github.com/swisspost/design-system/blob/fix/e2e-internet-header-with-storybook-v7/packages/internet-header/src/components/post-internet-header/post-internet-header.tsx#L185-L189

Other parts are loaded because `getLocalizedConfig` does the job of finding a default language: https://github.com/swisspost/design-system/blob/main/packages/internet-header/src/services/config.service.ts#L61

On the same topic, the external method `getCurrentLanguage` does have a fallback, but it's not spread to other parts of the component, so it can be wrong.
https://github.com/swisspost/design-system/blob/fix/e2e-internet-header-with-storybook-v7/packages/internet-header/src/components/post-internet-header/post-internet-header.tsx#L134-L136

If possible, we should extract the logic of `getUserLang` from `getLocalizedConfig`.

Reply:
I opened a new issue to handle this https://github.com/swisspost/design-system/issues/1865

Reply:
I'll close mine then ^^ I forgot to link to the comment.